.ci/windows/build.cmd

```diff
@@ -1,8 +1,13 @@
 set crypto=%1
 set builddir=%crypto%
+set args=
+
+if %TYPE% == cross (
+    set args=--cross-file .ci/cross/msvc/%ARCH:amd64_=%
```

Comment:
Ah, that's why you named the cross file `amd64_arm64`... It might be better if you could change the matrix for the msvc test target in `.github/workflows/test.yml` so it has separate `build_arch` and `host_arch` variables, but I'm not sure how that interacts with `ilammy/msvc-dev-cmd@v1`.

Reply:
Makes sense, it's cleaner this way.

We can use duplication instead of the conditional:

```yaml
  - name: Activate dev environment (cross)
    uses: ilammy/msvc-dev-cmd@v1
    with:
      arch: ${{ format('{0}_{1}', matrix.target.build, matrix.target.host) }}
    if: ${{ matrix.target.build != matrix.target.host }}

  - name: Activate dev environment (native)
    uses: ilammy/msvc-dev-cmd@v1
    with:
      arch: ${{ matrix.target.host }}
    if: ${{ matrix.target.build == matrix.target.host }}
```

I don't like copypasting stuff (too easy to forget one of them when updating in the future), so the expression is a bit obtuse.

`notest` instead of "boolean" `yes`/`no` to make short descriptions more obvious.